src/services/infinispanRefData.ts

```diff
@@ -226,6 +226,6 @@ export enum CounterType {
 }
 
 export enum CounterStorage {
-  VOLATILE = 'VOLATILE',
```

Comment:
@dvagnero We need to change filtering function if we change from all-caps to Capitalization in CounterStorage.

Do you recommend not using all-caps?